epic_capybara/cli/bara.py

```diff
@@ -54,14 +54,21 @@ def bara(files, match, unmatch, serve):
 
     for _file in files:
         tree = uproot.open(_file)["events"]
+
+        sort_by_evtnum = None
+        if "EventHeader/EventHeader.eventNumber" in tree.keys(recursive=True):
+            evtnum = tree["EventHeader/EventHeader.eventNumber"].array()
+            sort_by_evtnum = ak.argsort(ak.flatten(evtnum))
+
         keys = [
             key for key in tree.keys(recursive=True)
             if not key.startswith("PARAMETERS")
             and len(tree[key].branches) == 0
             and match_filter(key, match, unmatch)
         ]
+
         for key in keys:
-            arr.setdefault(key, {})[_file] = tree[key].array()
+            arr.setdefault(key, {})[_file] = tree[key].array(evtnum) if sort_by_evtnum is None else tree[key].array()[sort_by_evtnum]
```

Comment:
What does passing evtnum to array does here? Is the first argument of `TBranchElement.array()` going to the `interpretation` kwarg?

Reply:
Does nothing useful. Shouldn't be there.
```suggestion
            arr.setdefault(key, {})[_file] = tree[key].array() if sort_by_evtnum is None else tree[key].array()[sort_by_evtnum]
```